src/Call/Oneclick/InitPaymentRequest.php

```diff
@@ -45,11 +43,11 @@ public function __construct(
 		string $merchantId,
 		string $origPayId,
 		string $orderId,
-		int $totalAmount = null,
-		Currency $currency = null,
+		Cart $cart = null,
```

Comment:
@VasekPurchart mě to přijde významově nastejno. CSOB preferuje pro částku amount, tak jsem se toho držel. Máš jiný názor?